scripts/monitor/handlers/index.js

```diff
@@ -4,6 +4,9 @@ import { BlockchainEvent } from '../../../src/BlockchainEvent'
 const log = new Log('handlers')
 
 export async function store(eventData) {
+  if (eventData.length) {
```

Comment:
huh, wasn't aware this might happen. The docs say that the callback receives a result `object` https://github.com/ethereum/wiki/wiki/JavaScript-API#callback-return

That said I think is the responsibility of the StoreCli to report each event to the handler, meaning changing this line instead:

https://github.com/decentraland/marketplace/blob/master/scripts/monitor/StoreCli.js#L39

from:

```javascript
await handler(eventData)
// to something like 
if (eventData.length) await Promise.all(eventData.map(handler)
 else await handler(eventData)
```

what do you think?